package.json

```diff
@@ -19,7 +22,13 @@
     "url": "https://github.com/ethomson/probot-serverless-azurefunctions/issues"
   },
   "homepage": "https://github.com/ethomson/probot-serverless-azurefunctions#readme",
+  "devDependencies": {
+    "jest": "^26.6.1"
+  },
+  "peerDependencies": {
+    "probot": "~10.9.5"
+  },
   "dependencies": {
-    "probot": "^7.4.0"
+    "probot": "~10.9.5"
```

Comment:
I don't think you need `probot` in peerDependencies if you have it in `dependencies`. And `~10.9.5` would give you `10.9.6` which might break the imports of the internal helper methods. I'd suggest to change it to just `10.9.5`